rustls/src/crypto/aws_lc_rs/ticketer.rs

```diff
@@ -340,7 +340,10 @@ mod tests {
 
     #[test]
     fn ticketswitcher_switching_test() {
-        #[expect(deprecated)]
+        // XXX TODO SHOULD VERIFY THAT THIS IS OK: #[expect(deprecated)]
+        // QUICK WORKAROUND FOR RUST NIGHTLY: nightly-2024-05-01
+        // #[expect(deprecated)]
+        #[allow(deprecated)]
```

Comment:
I think it would be best to revert this & simply skip testing large feature set with nightly-2024-05-01, like the current situation.